src/functions-and-arrays.js

```diff
@@ -26,13 +68,37 @@ function sum() {}
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+function averageNumbers(numbersAvg) {
+  let sum1=0
+  let average=""
+  if(!numbersAvg.length){
+    return null;
+  }for (let i=0;i<numbersAvg.length;i++){
+  sum1+= numbersAvg[i]
+    average =sum1
+  
+      }
+      return average/numbersAvg.length;
+    };
+
 
 
 // Level 2: Array of strings
 const wordsArr = ['seat', 'correspond', 'linen', 'motif', 'hole', 'smell', 'smart', 'chaos', 'fuel', 'palace'];
 
-function averageWordLength() { }
+function averageWordLength(wordsArr) { 
```

Comment:
Here same thing, you can return the `total` divided by the `length` of the array `wordsArr` straight away:

```
function averageWordLength(wordsArr) {
  let total = 0;

  if (!wordsArr.length) {
    return null;
  }

  for (let i = 0; i < wordsArr.length; i++) {
    total += wordsArr[i].length;
  }

  return total / wordsArr.length;
}
```